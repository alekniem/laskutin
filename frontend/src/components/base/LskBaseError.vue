<template>
    <div class="red--text font-weight-bold">
        <slot />
    </div>
</template>

<script>
export default {
    name: "LskBaseError",
};
</script>
