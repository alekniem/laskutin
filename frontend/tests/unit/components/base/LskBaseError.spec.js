import LskBaseError from "@/components/base/LskBaseError";
import { createWrapper } from "UT/lsk_test_helper";

describe("LskBaseError", () => {
    it("should match snapshot", () => {
        const wrapper = createWrapper(LskBaseError);
        expect(wrapper.html()).toMatchSnapshot("no content in default slot");
    });

    it("should match snapshot", () => {
        const wrapper = createWrapper(LskBaseError, {
            slots: {
                default: "<div>DEFAULT</div>",
            },
        });
        expect(wrapper.html()).toMatchSnapshot("content in default slot");
    });
});
