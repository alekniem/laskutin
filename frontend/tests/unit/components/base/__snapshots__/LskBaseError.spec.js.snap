// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LskBaseError should match snapshot: content in default slot 1`] = `
<div class="red--text font-weight-bold">
  <div>
    DEFAULT
  </div>
</div>
`;

exports[`LskBaseError should match snapshot: no content in default slot 1`] = `
<div class="red--text font-weight-bold">
</div>
`;
